<ui version="4.0" >
 <class>ThreadRunnerDlg</class>
 <widget class="QDialog" name="ThreadRunnerDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Thread Runner</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="m_pCurrentL" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pPauseResumeB" >
       <property name="text" >
        <string>&amp;Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pAbortB" >
       <property name="text" >
        <string>&amp;Abort</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
